File: springboot-mustache/src/main/java/com/knowledgefactory/knowledgefactorydemo/HomeController.java
package com.knowledgefactory.knowledgefactorydemo;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class HomeController {

	@GetMapping({ "/", "/welcome" })
	public String hello(Model model) {

		return "index";
	}
}

File: kotlin-spring-main/kotlin-spring-thymeleaf-login-signup/src/main/kotlin/com/knf/dev/demo/kotlinspringthymeleafloginsignup/controller/HomeController.kt
package com.knf.dev.demo.kotlinspringthymeleafloginsignup.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class HomeController {
    @GetMapping("/login")
    fun login(): String {
        return "login"
    }

    @GetMapping("/")
    fun home(): String {
        return "index"
    }
}